validator/checker.py

```diff
@@ -207,10 +244,10 @@ def check_single_version_update(
             if insert:
                 assert asset_data.cryptocompare not in seen_elements, f'Duplicate cryptocompare {asset_data.cryptocompare}'
                 assert asset_data.coingecko not in seen_elements, f'Duplicate coingecko {asset_data.coingecko}'
-                if asset_data.name in seen_elements:
-                    logging.warning(f'Duplicate name {asset_data.name}')
-                if asset_data.symbol in seen_elements:
-                    logging.warning(f'Duplicate symbol {asset_data.symbol}')
+                if asset_data.name not in DUPLICATED_NAMES:
```

Comment:
Also add a comment here, why these symbol/names are duplicated. I am not even sure why. v1/v2 ?

Reply:
In V2 anyone can create a vault. Then there are some that are endorsed by yearn once they are battle tested. The information I used comes from the yearn v2 graph that tracks every yearn v2 vault, even the ones that are not endorsed. 

Since this is all the information provided thought that it was a good idea to save all of them. One thing I can do is filter to only get the ones in this list https://tokenlists.org/token-list?url=https://yearn.science/static/tokenlist.json

Reply:
So perhaps add a similar comment here? I think if a user uses a v2 vault we should detect it, even if it's not endorsed.

Reply:
Done